model/src/w3srcemd.F90

```diff
@@ -2097,7 +2145,7 @@ SUBROUTINE W3SRCE ( srce_call, IT, ISEA, JSEA, IX, IY, IMOD,          &
 #ifdef W3_IC3
         ATT=EXP(ICE*VDIC(IS)*DTG)
 #endif
-#ifdef W3_IC4
+#ifndef W3_IC4_ACCURATE_NUMERICS
```

Comment:
In a case of IC0 for example where there is no IC1-IC5 defined, then VDIC has not been defined (look at line 744 in this same script) and the build will give this error: This name does not have a type, and must have an explicit type.   [VDIC]
        ATT=EXP(ICE*VDIC(IS)*DTG)

Reply:
@erinethomas @NickSzapiro-NOAA do you agree with this change? If so could you please go ahead and make the changes?

Reply:
Keeping both makes sense to me to protect VDIC